### ISSUE TITLE
Please help me fix this error

### ISSUE THREAD

**Issue by Pang-0093** [justinpinkney/stable-diffusion#94]
Hi there, I use your code to finetune my customized SD (follow the instructions [example](https://github.com/LambdaLabsML/examples/tree/main/stable-diffusion-finetuning#fine-tuning-stable-diffusion:~:text=Fine%20Tuning%20Stable%20Diffusion))
But I got an error 
RuntimeError:
DistributedDataParallel is not needed when a module doesn't have any parameter that requires a gradient. 

I am confused and don't know why and how to fix it . Can someone help me out? Thanks a lot!

**Comment by Pang-0093** [comment]
turn outs to be that my model is loaded abnormally. No parameters require grads, then the error is raised
